modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -1507,6 +1507,7 @@ export class Wallet implements IWallet {
       reshare: params.reshare,
       message: params.message,
       disableEmail: params.disableEmail,
+      skipKeychain: (this.type() ?? 'hot') === 'hot' ? !!params.skipKeychain : undefined,
```

Comment:
Yea, after looking at the API, it can get a little messy if we only have this for hot-wallets. Gonna move this to a draft until I finalize the backend changes.

Reply:
Changed it to be dynamic based on the inputs. The inputs already handle `skipKeychain`, users using this method should have the expected outcome with the api while minimizing integration changes clients need to make.